_topic_maps/_topic_map_rosa.yml

```diff
@@ -137,12 +152,10 @@ Topics:
     File: rosa-creating-cluster
 # - Name: Creating a ROSA cluster using the web console
 #   File: rosa-creating-cluster-console
-  - Name: Creating an AWS PrivateLink cluster on ROSA
-    File: rosa-aws-privatelink-creating-cluster
-  - Name: Accessing a ROSA cluster
-    File: rosa-accessing-cluster
-  - Name: Configuring identity providers using the Red Hat OpenShift Cluster Manager
-    File: rosa-config-identity-providers
+#  - Name: Accessing a ROSA cluster
+#    File: rosa-accessing-cluster
+#  - Name: Configuring identity providers using the Red Hat OpenShift Cluster Manager
+#    File: rosa-config-identity-providers
   - Name: Deleting access to a ROSA cluster
     File: rosa-deleting-access-cluster
   - Name: Deleting a ROSA cluster
```

Comment:
Does the _Command quick reference for creating clusters and users_ apply to STS as well? If so, should it be moved up a directory level?